### PR TITLE
Mega-form

### LINKED ISSUE #37: Create Sequence Diagram for mega-form structure
Show a sequence of form elements to be filled during the transportation following a sequence of timeline.

### LINKED ISSUE #38: Mega-form Content
- [x] It should contain all information along with rules & regulations present on each breakdown form/document.

- [x] It should highlight overlapped information and form elements

### PR DESCRIPTION
**Pull request by SanaKhan1997**
Contains merged information from CFIA forms i.e. Animal Record, FWR, Transfer of Care and Contingency Plan

### CODE REVIEW

**Review by holtzmak — COMMENTED** [review]
Nice to see all the form elements together. Could you explain to me how to read the overlap flags you added? For example:

_Receiver’s Contact number in case of emergency:_ __________CP_

Is it just this line in particular that overlaps with the Contingency Plan, what about the other elements near it like:

_PID: Name:_
_Address:____________

and 

_Date of arrival (dd/mm/yyyy):____________
_Time of arrival:_______ _AM/PM_______
_Arrival: All animals arrived in good condition ? Yes/No_

?

**Comment by SanaKhan1997** [comment]
I've organized information according to heading given i.e.

1. Shipper's Info
2. Transporter's Info
3. Receiver's Info
4. FWR
5. Loading Info
6. Contingency Plan

If a color tag is beside a form element it is only specific to that form element and does not affect anything around. But if a tag is beside a heading(one from above), it covers every form element in that heading.

**Review by holtzmak — APPROVED** [review]
Thanks for clearing up my confusion, I had originally thought that was how we should read the document but wanted to be sure. Before merging this, there's a few more things I'm wondering about (that you may or may not want to do):

1. Maybe we shouldn't call it the "mega-form" in the document itself, we the team know what "mega-form" means but we should probably call it what it is: "Animal Transport Record Form with Overlap Signifiers" (or similar, simpler).
2. Should the form be in the same color format as our other documentation files in the documentation folder? e.g. the other forms have orange, our names on the documentation, a left-aligned header. Tim had mentioned that he likes attention to detail. Do you think we could do that (maybe too nitpicky)?
3. I would change the file name to match your decision against # 1

**Comment by SanaKhan1997** [comment]
Alright! I'm going to change the document template to the one we've been using and for the name part how does "Animal Transportation Form" sound and I can specify CFIA forms it contains in the description.

**Comment by holtzmak** [comment]
> Alright! I'm going to change the document template to the one we've been using and for the name part how does "Animal Transportation Form" sound and I can specify CFIA forms it contains in the description.

That's perfect

**Comment by SanaKhan1997** [comment]
Here's the updated mega-form [Animal Transportation Form](https://github.com/holtzmak/Humane-Transport/blob/SanaKhan1997-patch-1/documentation/Animal%20Transportation%20Form-(Mega%20Form).pdf)
@holtzmak

**Comment by holtzmak** [comment]
@SanaKhan1997 The only thing worth noting is that the "Animals Loaded" Table appears cut-off in the PDF.

File: documentation/Megaform Sequence Diagram.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (holtzmak) [inline documentation/Megaform Sequence Diagram.svg]: What is this? Does this need to be under revision control?

Comment (holtzmak) [reply documentation/Megaform Sequence Diagram.svg]: I see now that this is the sequence diagram drawing. Can we make it a file type other than svg? This file type is a strange one to put under revision control (plus, it's basically one super long line so we should at least format it to be readable)

Comment (SanaKhan1997) [reply documentation/Megaform Sequence Diagram.svg]: I was planning to put the image in the document template and upload it as pdf if that's fine.

Comment (holtzmak) [reply documentation/Megaform Sequence Diagram.svg]: Yeah, that's what I'd expected, too

**Pull request merged**